src/main/java/VASSAL/chat/ui/RoomTreeRenderer.java

```diff
@@ -60,8 +60,8 @@ public Component getTreeCellRendererComponent(
     boolean expanded,
     boolean leaf,
     int row,
-    boolean hasFocus)
-  {
+    boolean hasFocus) {
```

Comment:
This can't be enforced with checkstyle, it is either "LeftCurly on new line" or "LeftCurly on EOL".

Exceptions like a method definition with a long arguments list should generally not exist, otherwise this would be valid, and we would be back at C-style for an arbitrary amount of method definitions:
```
public void method(int arg1,
  int arg2)
{
```

The whole LeftCurly check could be disabled for method definitions but that would defeat its purpose.